packages/@poupe-vue/src/test-setup.ts

```
@@ -0,0 +1,36 @@
+import { beforeAll } from 'vitest';
+
+// Suppress Vue warnings in tests
+beforeAll(() => {
+  // Store original console methods
+  const originalWarn = console.warn;
+  const originalError = console.error;
+
+  // Suppress Vue warnings that are expected in tests
+  console.warn = (...args) => {
```

ESLint

[error] The variable `args` should be named `arguments_`. A more descriptive name will do too.

(unicorn/prevent-abbreviations)